online-banking-cz/src/main/java/com/adyen/checkout/onlinebankingcz/OnlineBankingCZView.kt

```diff
@@ -109,6 +114,29 @@ class OnlineBankingCZView @JvmOverloads constructor(
         binding.textInputLayoutOnlineBanking.isEnabled = enabled
     }
 
+    private fun styleTermsAndConditions(string: String) {
+        val firstIndex = string.indexOf("%#", 0, ignoreCase = true)
+        val secondIndex = string.indexOf("%#", firstIndex + 2, ignoreCase = true)
+
+        val newString = string.replace("%#", "", ignoreCase = true)
```

Comment:
We're only replacing `%#` with `""`, should we also replace `#%`?

Reply:
the string doesn't contain #% so I don't think we need that

Reply:
My bad 🤦 I assumed they open with `%#` and close with `#%`